osaca/parser/parser_AArch64.py

```diff
@@ -326,6 +350,8 @@ def process_memory_address(self, memory_address):
         """Post-process memory address operand"""
         # Remove unnecessarily created dictionary entries during parsing
         offset = None if 'offset' not in memory_address else memory_address['offset']
```

Comment:
better alternative:
`offset = memory_address.get('offset', None)`

also applies to base and index lines below

Reply:
Resolved in 7e7269c